curriculum/challenges/english/25-front-end-development/workshop-linked-list-class/68867c3898b09115d8c5140f.md

```diff
@@ -0,0 +1,55 @@
+---
+id: 68867c3898b09115d8c5140f
+title: Step 14
+challengeType: 20
+dashedName: step-14
+---
+
+# --description--
+
+In the `else` block, right after the `while` block, assign `node` to `current_node.next`.
+
+# --hints--
+
+You should assign `node` to `current_node.next` in the `else` block.
```

Comment:
are you sure this is the right test for this step? it does not match with description, and the statement is already there